### PR TITLE
Feature/dynamic index meta path

### PR DESCRIPTION
**Pull request by Cyclonecode**
Store temporary files generated from delta indexer in separate `tmp` folder in order to not clog up the src folder. The entire `tmp` folder is also ignored.

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ let config = require('./config');
 let logger = require('./log');
 let factory = new AdapterFactory(config);
 const jsonFile = require('jsonfile')
-const INDEX_META_PATH = process.env.INDEX_META_PATH ? process.env.INDEX_META_PATH : '.lastIndex.json'
+const INDEX_META_PATH = process.env.INDEX_META_PATH ? './tmp/' + process.env.INDEX_META_PATH : './tmp/.lastIndex.json'
```
Comment (pkarw) [inline src/cli.js]: Will it work if ./tmp doesn’t exists (first run)?

**Comment by Cyclonecode** [comment]
I will try to look on this during this week =)

**Comment by Cyclonecode** [comment]
@pkarw - I've added a check to see if the tmp folder exists.

**Comment by Cyclonecode** [comment]
@pkarw - Good catch, now it should be fine I think.

**Comment by Cyclonecode** [comment]
I don't remember why i removed it? Re-added the node_modules to gitignore now.

**Comment by pkarw** [comment]
Thanks!

**Pull request merged**